packages/tabs-overflow/src/TabsOverflow.ts

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    PropertyValues,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import {
+    query,
+    queryAssignedElements,
+    state,
+} from '@spectrum-web-components/base/src/decorators.js';
+import { classMap } from '@spectrum-web-components/base/src/directives.js';
+import { ResizeController } from '@lit-labs/observers/resize_controller.js';
+import { Tabs } from '@spectrum-web-components/tabs';
+import '@spectrum-web-components/action-button/sp-action-button.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-left.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-right.js';
```

Comment:
You can probably get away with just importing on of these and using CSS transforms to flip. Also not sure if these should be coming from `@spectrum-web-components/icons-ui` following practice from`sp-accordion`. You'll also want to list the package as a dependency